### PR TITLE
feature/SKOOP-138

### PR DESCRIPTION
**Pull request by LebedkoDmitry**
Changelogs for schema/index initialization and demo data initialization were separated and placed in the different files.
The property --liquigraph.change-log=${file} is necessary to be used to choose between changelog files.
@georgwittberger could you please take a look at the changes?

### CODE REVIEW

**Comment by LebedkoDmitry** [comment]
> The Dockerfile should not be modified here. The environment variable can be passed on deployment in the Docker Compose configuration just like any other application setting (e.g. Neo4j database URI).

Thank you. Done.

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 FROM openjdk:11
 LABEL maintainer="georg.wittberger@gmail.com"
 ARG JAR_FILE
+ARG LIQUIGRAPH_CHANGELOG
```
Comment (georgwittberger) [inline Dockerfile]: Please just revert these changes.

Comment (LebedkoDmitry) [reply Dockerfile]: I didn't know Liquigraph respect the variable out-of-the-box.
I reverted the changes. Thanks.
